src/crypto/tls/common.go

```diff
@@ -420,6 +444,10 @@ type ClientHelloInfo struct {
 	// might be rejected if used.
 	SupportedVersions []uint16
 
+	// SupportDelegatedCredential is true if the client indicated willingness
+	// to negotiate the delegated credential extension.
+	SupportDelegatedCredential bool
```

Comment:
Nitpick: `SupportsDelegatedCredential` or `SupportsDelegatedCredentials` might be better, although I'll defer judgement to a native speaker.